Vungle/VungleAdapter/ALVungleMediationAdapter.h

```diff
@@ -10,7 +10,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ALVungleMediationAdapter : ALMediationAdapter<MASignalProvider, MAInterstitialAdapter, MARewardedAdapter, MAAdViewAdapter, MANativeAdAdapter>
+@interface ALVungleMediationAdapter : ALMediationAdapter<MASignalProvider, MAInterstitialAdapter, /* MAAppOpenAdapter */ MARewardedAdapter, MAAdViewAdapter, MANativeAdAdapter>
```

Comment:
Why's the new adapter commented out? Add a comma too

Reply:
Have the same question, might need to add `// TODO:` here.

Reply:
@YueVungle will update it once 6.12 is updated so. no todos here. 👍